frontend/src/components/TodoTask/IconButton/IconButton.tsx

```diff
@@ -0,0 +1,35 @@
+import { Button } from 'antd';
+import { StarFilled, StarOutlined, BorderOutlined, CheckSquareOutlined } from '@ant-design/icons';
+import { blue } from '@ant-design/colors';
+
+export type IconButtonProps = {
+  onClick(): void;
+  color?: string;
```

Comment:
Fajnie jakbyśmy zamiast koloru ustawiali czy jest to kolor "primary | secondary | warning" itp. Albo coś w tym stylu, tak, aby zmiana w pliku s głównym tematem, powodowała zmiany w  tych wszystkich miejscach :)

Reply:
usunąłem tego propa